package.json

```diff
@@ -14,8 +18,8 @@
     "sass-loader": "^1.0.2",
     "style-loader": "^0.12.3",
     "virtual-dom-loader": "^0.1.0",
-    "webpack": "^1.10.1",
-    "webpack-dev-server": "^1.10.1"
+    "webpack": "^1.8.4",
```

Comment:
Is there a reason for downgrading webpack or is it just accidental? Just curious.